ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -436,13 +436,19 @@ Wait For DataScienceCluster CustomResource To Be Ready
 
 Component Should Be Enabled
     [Arguments]    ${component}    ${dsc_name}=${DSC_NAME}
-    ${status} =    Is Component Enabled    ${component}    ${dsc_name}
-    IF    '${status}' != 'true'    Fail
+    ${status} =   Set Variable   False
+    WHILE   '${status}' != 'true'    limit=60 seconds
```

Comment:
One concern I have is that doing this in a loop will hammer the cluster with a lot of requests for 1 minute in case the component is not enabled, should we maybe add a short sleep to the loop body?

Reply:
mmm we have not had issues with hammering the cluster AFAIK